README.md

```diff
@@ -63,7 +63,7 @@ Crypto Watcher is a webapp that displays information about the top 100 cryptocur
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Live Demo : [Crypto Watcher](...)
+> Live Demo : [Crypto Watcher](https://gilberthappi.github.io/React-capstone-project---Metrics-webapp/)
 
```

Comment:
- [ ] The PR should have all changes that have been made to this project, This PR has only some changes to the README file. Please fix that and submit a PR link that has all changes made.

- [ ] Please add a properly deployed link to your app that actually works, not showing the README file as you did.
- [ ] Please add a link here in the README file for the live demo presentation of your app as required [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=of%20the%20project%2C-,you%20should%20record%20a%20video%20presenting%20the%20features%20of%20the%20project%20you%20built,-.%20It%20is%20a).
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=requires%20that%20you-,give%20appropriate%20credit%20to%20the%20author,-.%20Therefore%2C%20you%20must), Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) here in the "Acknowledgements" section.